lib/coverband/base.rb

```diff
@@ -42,6 +42,7 @@ def reset_instance
       @ignore_patterns = Coverband.configuration.ignore
       @sample_percentage = Coverband.configuration.percentage
       @reporter = Coverband::RedisStore.new(Coverband.configuration.redis) if Coverband.configuration.redis
+      @reporter = Coverband.configuration.reporter if Coverband.configuration.reporter
```

Comment:
I'd rather merge these two lines, so `if reporter set it, else if redis, set it to the redis-store`